unified-scheduler-pool/src/lib.rs

```diff
@@ -2108,6 +2145,14 @@ impl<S: SpawnableScheduler<TH>, TH: TaskHandler> ThreadManager<S, TH> {
                     session_result_sender
                         .send(result_with_timings)
                         .expect("always outlived receiver");
+
+                    if matches!(scheduling_mode, BlockProduction) {
+                        datapoint_info!(
+                            "unified_scheduler-bp_session_stats",
+                            ("slot", current_slot.unwrap_or_default() as i64, i64),
+                            ("block_size_estimate", block_size_estimate as i64, i64),
```

Comment:
nit: remove the `as i64`. the macro does that internally 😉

Reply:
til: d728410672ac685e42db6879c423a85a2ae49e7f